packages/ovm/package.json

```diff
@@ -57,6 +57,7 @@
     "ethereum-waffle": "2.1.0",
     "ethers": "^4.0.37",
     "ethlint": "^1.2.5",
+    "lodash": "^4.17.15",
```

Comment:
I kiiinda am a fan of not requiring `lodash` because it technically doesn't have to be used & adds weight to the package. That said, I also do really enjoy programming with lodash. Maybe if we only used lodash for testing functions & added it only to `devDependencies` then it'd be a no brainer? Either way, this is not a strong opinion so I'm personally fine with keeping lodash usage exactly as is.

Reply:
Yeah agreed we shouldn't add dependencies unless their benefit outweighs the cost. I was using native `reduce` but `fromPairs` made it a bit more readable. I [moved it to a `devDependency`](https://github.com/ethereum-optimism/optimism-monorepo/pull/25/commits/a7ffb92783acde987a1517f997c3ca118948da0a) so we can punt on that decision for now. I also added it to a list of things we can discuss as a team while we're all in person next week! 😄  -> [TypeScript/Solidity Style Guide Discussion](https://hackmd.io/sO2Fk6x9Tt6QyQSnjdo5Zg?view)